### DISCUSSION TITLE
Can't verify two step verification from GitHub App

### DISCUSSION THREAD

**Discussion by orzklv** [community/community#52014] [Mobile]
### Select Topic Area

Bug

### Body

While trying to verify from iOS GitHub App, it throws error:

`The operation couldn't be completed.
(CryptoTokenKit error -3.)
`

It's been like 3 weeks since that error started popping up and up to this time I've been using 1Password's 2ns step pin generation to proceed with `sudo mode`.

**Comment by stevepopovich** [comment]
Hi @katsuki-yuri sorry to hear that is happening. Do you have any screenshots of the error? 

Regardless, I will pass this on to the team for investigation. Thanks for writing in!

**Comment by stevepopovich** [comment]
Hi again, do you mind trying a reinstall of the app and see if that helps?

**Reply by orzklv** [reply]
Hi @stevepopovich, I do apologize for late response. I resolved the issue by reinstalling the app. Thanks for your attention 🙏🏻

**Reply by stevepopovich** [reply]
Great, thanks!

**Reply by KenBirman** [reply]
I cleared the cache, reinstalled the app and signed back in, no problem, but got the same error again anyhow.  So this is a current issue...   Below @dmythro says "sign out of all accounts", unsure what that would be beyond what I did.  Anyhow, any ideas?

**Reply by LordNex** [reply]
I'm having the same problem. Reinstalling the app didn't solve the problem. Any suggestions?

**Reply by LordNex** [reply]
> Hi @stevepopovich, I do apologize for late response. I resolved the issue by reinstalling the app. Thanks for your attention 🙏🏻

This isn't working. Any suggestions? I've reinstalled the app but still get the crypto token error.

**Comment by dmythro** [comment]
Hi, this error is pretty odd. Got it recently.
Reinstalling app didn't help. After reinstall I was already signed in and got the same error on code approval.

**Reply by dmythro** [reply]
UPD: Clear cache, sign out from all accounts, reinstall the app and then it seems fixed.

**Comment by DRingOfficial** [comment]
Ok

**Reply by edmacdonald** [reply]
I'm getting this now, 2 years later :/

**Comment by stevepopovich** [comment]
I have filed a bug for this internally.

**Comment by Shuddown** [comment]
This has been affecting me too. I'm being forced to use my old android phone for GitHub...

**Comment by Junjun13910** [comment]
Im done stressing

**Comment by stefan22** [comment]
In my case, I fixed this same error by disabling the AntiVirus on my iPhone, and trying again.

**Comment by Junjun13910** [comment]
Thank you sir for your update I well try it later



Adrian

**Comment by lambegraham** [comment]
Also having the same myself when trying to verify login. iOS 18.5

**Comment by perera2k4** [comment]
I had the same problem for android 15

**Reply by bluebeckie** [reply]
In my case,no notification from my GitHub app at all.

**Reply by NBKRedSpy** [reply]
On my iPhone, I have to close the app and open it back up to get the prompt for the number.

**Comment by jor2** [comment]
Also having this issue with iPhone.

**Comment by NBKRedSpy** [comment]
On iOS as well.  Oddly, after uninstalling the app and reinstalling it, my account was still logged in.
And yes, this was "delete" and not "remove from home screen" ;)

**Comment by dacort** [comment]
Ran into this on iPhone as well. Didn't uninstall the app, just signed out and back in and it worked. Tried clearing the cache in the app settings first, but that did nothing.

**Comment by markedup** [comment]
Had the same issue on iOS 18 and still on iOS 26 after upgrading. This is how I fixed it...

> [!IMPORTANT]
> Ensure you have another 2FA method setup in [your GitHub account security](https://github.com/settings/security) - like Passkey or Authenticator - before you proceed, otherwise you may end up locked out of your account.

1. Open the GitHub app.
2. Pick **Profile** from the navigation bar along the bottom.
3. Select the cog (⚙️) in the top-right.
4. Scroll down and select the **Clear cache** option.
5. Just above, select the **Manage Accounts** option.
6. Select **Edit** in the top-right and then select **Sign Out All Accounts**.
7. Force-close the app (swipe up and hold from the bottom of the screen, then swipe GitHub up off the screen to force-close it).
8. Start the GitHub app again and sign in as normal.

2FA login prompts should now work for you again.

**Reply by SiboVG** [reply]
Thanks, that worked (iOS 26)!

**Comment by gnunesdev** [comment]
I've been struggling with the same error. Unfortunately I dont have any other 2fa set up, so it's impossible to login in the web app. 

Considering that it seems to be an old problem, is there any investigation on it?

**Reply by lambegraham** [reply]
> I've been struggling with the same error. Unfortunately I dont have any other 2fa set up, so it's impossible to login in the web app. 
> 
> Considering that it seems to be a old problem, is there any investigation on it?

Sign out of app and sign back in

**Reply by gnunesdev** [reply]
> > I've been struggling with the same error. Unfortunately I dont have any other 2fa set up, so it's impossible to login in the web app. 
> > 
> > Considering that it seems to be a old problem, is there any investigation on it?
> 
> Sign out of app and sign back in

I dont have any other 2fa, so if I sign out I dont think I'll be able to sign back in 😔

**Comment by NBKRedSpy** [comment]
Clearing the cache and logging off fixed the issue for me. 
I still have to close the app and open it back up to actually see the code, but that has been an issue for a while.

Edit:  A long time ago the app would actually show the code when already opened.

**Comment by saper** [comment]
December 2025, GitHub Mobile Version 1.236.0 (179029796) on iOS 18.4, worked around by using another factor.

**Comment by Szymczek** [comment]
same issue

**Comment by giovariot** [comment]
same here

**Comment by ozgurg** [comment]
I tried uninstalling it. After reinstalling, the previously logged-in account was still logged in and it wasn't working at all. I signed out of all accounts, uninstalled, reinstalled, and logged in again. Now it works.
